<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui_Profiles</class>
 <widget class="QDialog" name="gui_Profiles">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="3">
    <widget class="QLineEdit" name="ctrl_profileName"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lbl_profileDescription_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPlainTextEdit" name="ctrl_profileDescription"/>
   </item>
   <item row="1" column="0" rowspan="4" colspan="3">
    <widget class="QListWidget" name="ctrl_profileList">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_profileList">
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbl_profileName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ctrl_profileClose">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	margin: 0px;
    padding: 0px;
	image: url(:/Images/icon_close.png)
}

QPushButton:hover {
	border: none;
	margin: 0px;
    padding: 0px;
	image: url(:/Images/icon_close_lite.png)
}

QPushButton:disabled {
	border: none;
	margin: 0px;
    padding: 0px;
	image: url(:/Images/icon_close_lite.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="ctrl_LoadProfile">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ctrl_SaveProfile">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ctrl_EraseProfile">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_profileDescription">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QTextBrowser" name="ctrl_profilePreview"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
